### PR TITLE
00 init

### PR DESCRIPTION
**Pull request by llaulier**

### CODE REVIEW

**Review by RoxaneDrelon — DISMISSED** [review]
So no evill

File: sql/capsule.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+-- Query: SELECT * FROM yooviz_db.capsule
+LIMIT 0, 1000
+
+-- Date: 2020-06-22 17:52
+*/
+INSERT INTO capsule
+    (`capsule_name`,`description
+
+
+
```
Comment (LoicBrassart) [inline sql/capsule.sql]: ... Why these empty lines ?

File: sql/poi.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/*
+-- Query: SELECT * FROM yooviz_db.poi
+LIMIT 0, 1000
+
+-- Date: 2020-06-22 17:56
+*/
+INSERT INTO poi
+    (`id`,`poi_name
+
```
Comment (LoicBrassart) [inline sql/poi.sql]: Useless empty line

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Please remove empty lines

**Review by RoxaneDrelon — APPROVED** [review]
Clean indentation

**Review by LoicBrassart — APPROVED** [review]
SnE!

**Pull request merged**